docs/components/modals/index.html

```diff
@@ -42,22 +42,151 @@ <h3>Demo Modal Header</h3>
+              jelly beans muffin pie sugar plum gummies. Pudding dessert
+              macaroon jelly-o pastry topping brownie gingerbread. Icing
+              biscuit tiramisu chocolate bar lemon drops brownie I love
+              caramels. Biscuit muffin cotton candy pastry cotton candy sweet.
+            </p>
+
+            <p>
+              Ice cream cheesecake jujubes. Sugar plum brownie biscuit powder toffee
+              halvah brownie. Donut lollipop bonbon. Wafer cotton candy marshmallow
+              macaroon souffle toffee. Fruitcake sugar plum souffle biscuit. Pie
+              wafer pastry jujubes brownie ice cream donut cake. Cheesecake chupa
+              chups apple pie pie sesame snaps chocolate.
+            </p>
+
+            <p>
+              I love sweet roll I love sugar plum pie. Toffee lemon drops
+              cotton candy cake gummi bears chocolate bar. Tart chocolate
+              cotton candy halvah oat cake jelly beans. Jelly tart pudding
+              biscuit pastry dragee topping caramels muffin. Pastry pudding
+              oat cake chocolate bar. Candy tart croissant. I love icing candy
+              croissant I love carrot cake.
+            </p>
+
+            <p>
+              Powder halvah candy cookie bear claw donut fruitcake brownie.
+              Lemon drops I love gingerbread carrot cake oat cake fruitcake
+              brownie gummies fruitcake. Liquorice gingerbread powder sweet.
+              Marzipan cheesecake cake. Candy cupcake bonbon carrot cake chupa
+              chups pie pie. Donut gummies I love lemon drops gingerbread
+              powder candy topping caramels. Pie toffee croissant jelly-o.
+            </p>
+          </hx-div>
+          <footer>
+            <button class="hxBtn hxPrimary">Confirm</button>
+            <button class="hxBtn">Cancel</button>
+          </footer>
+        </hx-modal>
+      </div>
+
+      <footer>
+        {% code 'html' %}
+          <hx-disclosure aria-controls="scroll-modal" class="hxBtn">
+            Open Modal
+          </hx-disclosure>
+
+          <hx-modal id="scroll-modal">
+            <header>
+              <h3>Demo Modal Header</h3>
+            </header>
+            <hx-div scroll="vertical">
+              <p>...</p>
+              <p>...</p>
+              <p>...</p>
+              <p>...</p>
+              <p>...</p>
+              <p>...</p>
+              <p>...</p>
+              <p>...</p>
+            </hx-div>
+            <footer>
+              <button class="hxBtn hxPrimary">Confirm</button>
+              <button class="hxBtn">Cancel</button>
+            </footer>
+          </hx-modal>
+        {% endcode %}
+      </footer>
+    </div>
+    <p class="hxSubdued hxSubBody">
+      <hx-icon type="info-circle"></hx-icon>
+      Please refer to <code><a href="components/box/#scrolling-containers">
+      scrolling containers</a></code> documentation for scrolling modal body.
```

Comment:
Maybe update the link to show...

```
Please refer to "Scrolling Containers" documentation for more information about scrolling modal content.
```
(`"Scrolling Containers"` would be the link)